portfolio/src/main/webapp/script.js

```diff
@@ -12,29 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Computer Science', 'Business', 'Design', 'Marketing'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getFacts() {
+  fetch('/data').then(response => response.json()).then((fun_facts) => {
+    
+    const fun_factsListElement = document.getElementById('facts-container');
+    fun_factsListElement.innerHTML = '';
+    fun_factsListElement.appendChild(
+        createListElement('Fact One: ' + fun_facts.factOne));
+    fun_factsListElement.appendChild(
+        createListElement('Fact Two: ' + fun_facts.factTwo));
+    fun_factsListElement.appendChild(
+        createListElement('Fact Three: ' + fun_facts.factThree));
+  });
```

Comment:
The funFacts could be returned as a json Array. In that case, you could just use a for loop here. Remaining steps might require to change this to that array to return an array of comments. So ok, if you want do it in a follow up PR if you prefer. 

 https://www.w3schools.com/js/js_json_arrays.asp

Reply:
Similar to the comment above - I'll leave as is and edit it for the next PR (Done)